.travis.yml

```diff
@@ -1,10 +1,9 @@
-language: java
 sudo: false
+language: java
 
 jdk:
-  - openjdk8
   - oraclejdk8
-  - oraclejdk9
```

Comment:
This is no longer supported in Travis https://travis-ci.community/t/oraclejdk9-broken/3484